### ISSUE TITLE
Files uploaded to LFS via git-as-svn can be downloaded by git-as-svn only, not Git.

### ISSUE THREAD

**Issue by aologos** [bozaro/git-as-svn#130]
I have a gitlab server with LFS and git-as-svn installed. If I upload a large file via Git, the file can be downloaded via Git and SVN. If I upload the large file via SVN, the file can be downloaded via SVN only. When I try to perform git pull, I get following error:
```
a@ACER-E3-112-C6XG MINGW64 ~/Documents/test (master)
$ GIT_CURL_VERBOSE=1 GIT_TRACE=1 git pull
16:03:28.939196 git.c:371               trace: built-in: git 'pull'
16:03:28.957834 run-command.c:369       trace: run_command: 'fetch' '--update-head-ok'
16:03:28.961340 git.c:371               trace: built-in: git 'fetch' '--update-head-ok'
16:03:28.976973 run-command.c:369       trace: run_command: 'git-remote-http' 'origin' 'http://ServerGitlabHere:8888/test/test.git'
* Couldn't find host ServerGitlabHere in the _netrc file; using defaults
* timeout on name lookup is not supported
*   Trying 192.168.0.10...
* TCP_NODELAY set
* Connected to ServerGitlabHere (192.168.0.10) port 8888 (#0)
> GET /test/test.git/info/refs?service=git-upload-pack HTTP/1.1
Host: ServerGitlabHere:8888
User-Agent: git/2.12.2.windows.1
Accept: */*
Accept-Encoding: gzip
Pragma: no-cache
 
< HTTP/1.1 401 Unauthorized
< Server: nginx
< Date: Tue, 15 Aug 2017 06:03:20 GMT
< Content-Type: text/plain; charset=utf-8
< Content-Length: 26
< Connection: keep-alive
< Cache-Control: no-cache
< WWW-Authenticate: Basic realm="GitLab"
< X-Content-Type-Options: nosniff
< X-Frame-Options: DENY
< X-Request-Id: 332705fb-a79b-4761-9cda-2b76906fd247
< X-Runtime: 0.056764
< X-Ua-Compatible: IE=edge
< X-Xss-Protection: 1; mode=block
<
* Connection #0 to host ServerGitlabHere left intact
16:03:29.092492 run-command.c:369       trace: run_command: 'git credential-manager get'
16:03:29.162014 git.c:596               trace: exec: 'git-credential-manager' 'get'
16:03:29.162014 run-command.c:369       trace: run_command: 'git-credential-manager' 'get'
* Couldn't find host ServerGitlabHere in the _netrc file; using defaults
* Found bundle for host ServerGitlabHere: 0x1219990 [can pipeline]
* Re-using existing connection! (#0) with host ServerGitlabHere
* Connected to ServerGitlabHere (192.168.0.10) port 8888 (#0)
* Server auth using Basic with user 'UsernameHere'
> GET /test/test.git/info/refs?service=git-upload-pack HTTP/1.1
Host: ServerGitlabHere:8888
Authorization: Basic YXRhcmFzb3Y6aGosam43MmZ1Zm4=
User-Agent: git/2.12.2.windows.1
Accept: */*
Accept-Encoding: gzip
Pragma: no-cache
 
< HTTP/1.1 200 OK
< Server: nginx
< Date: Tue, 15 Aug 2017 06:03:21 GMT
< Content-Type: application/x-git-upload-pack-advertisement
< Content-Length: 422
< Connection: keep-alive
< Cache-Control: no-cache
< Strict-Transport-Security: max-age=31536000
<
* Connection #0 to host ServerGitlabHere left intact
16:03:29.669580 run-command.c:369       trace: run_command: 'git credential-manager store'
16:03:29.754186 git.c:596               trace: exec: 'git-credential-manager' 'store'
16:03:29.754186 run-command.c:369       trace: run_command: 'git-credential-manager' 'store'
16:03:30.122541 run-command.c:369       trace: run_command: 'rev-list' '--objects' '--stdin' '--not' '--all' '--quiet'
16:03:30.155048 run-command.c:369       trace: run_command: 'rev-list' '--objects' '--stdin' '--not' '--all' '--quiet'
16:03:30.170674 git.c:371               trace: built-in: git 'rev-list' '--objects' '--stdin' '--not' '--all' '--quiet'
16:03:30.201925 run-command.c:1155      run_processes_parallel: preparing to run up to 1 tasks
16:03:30.201925 run-command.c:1187      run_processes_parallel: done
16:03:30.201925 run-command.c:369       trace: run_command: 'gc' '--auto'
16:03:30.223565 git.c:371               trace: built-in: git 'gc' '--auto'
16:03:30.239476 run-command.c:369       trace: run_command: 'merge' 'FETCH_HEAD'
16:03:30.255103 git.c:371               trace: built-in: git 'merge' 'FETCH_HEAD'
Updating 348acb4..9b1295d
16:03:30.270730 run-command.c:369       trace: run_command: 'git-lfs filter-process'
trace git-lfs: run_command: 'git' version
trace git-lfs: run_command: 'git' config -l
trace git-lfs: Initialize filter-process
trace git-lfs: Read filter-process request.
Downloading Crash_Steel_Pipe_Rolling.wav (289.92 KB)
trace git-lfs: tq: running as batched queue, batch size of 100
trace git-lfs: tq: sending batch of size 1
trace git-lfs: api: batch 1 files
trace git-lfs: Filled credentials for http://ServerGitlabHere:8888/test/test.git/info/lfs
trace git-lfs: HTTP: POST http://ServerGitlabHere:8888/test/test.git/info/lfs/objects/batch
> POST /test/test.git/info/lfs/objects/batch HTTP/1.1
> Host: ServerGitlabHere:8888
> Accept: application/vnd.git-lfs+json; charset=utf-8
> Authorization: Basic * * * * *
> Content-Length: 125
> Content-Type: application/vnd.git-lfs+json; charset=utf-8
> User-Agent: git-lfs/2.0.1 (GitHub; windows amd64; go 1.8; git 678cdbd4)
>
{"operation":"download","objects":[{"oid":"891a4bd6cf8e6fb0dfc716f7a31219c970dc55d8220e7570b42b328e7edebc9f","size":296878}]}trace git-lfs: HTTP: 200
 
< HTTP/1.1 200 OK
< Transfer-Encoding: chunked
< Cache-Control: max-age=0, private, must-revalidate
< Connection: keep-alive
< Content-Type: application/json; charset=utf-8
< Date: Tue, 15 Aug 2017 06:03:22 GMT
< Etag: W/"528488322ebb46c255ee05cd1e270c79"
< Server: nginx
< Strict-Transport-Security: max-age=31536000
< X-Content-Type-Options: nosniff
< X-Frame-Options: DENY
< X-Request-Id: 49fe64b6-9a1b-4657-b131-3c467029fe65
< X-Runtime: 0.132491
< X-Ua-Compatible: IE=edge
< X-Xss-Protection: 1; mode=block
<
trace git-lfs: HTTP: {"objects":[{"oid":"891a4bd6cf8e6fb0dfc716f7a31219c970dc55d8220e7570b42b328e7edebc9f","size":296878,"error":{"code":404,"message":"Object does not exist on the server or you don't have permissions to access it"}}]}
{"objects":[{"oid":"891a4bd6cf8e6fb0dfc716f7a31219c970dc55d8220e7570b42b328e7edebc9f","size":296878,"error":{"code":404,"message":"Object does not exist on the server or you don't have permissions to access it"}}]}trace git-lfs: tq: starting transfer adapter "basic"
trace git-lfs: xfer: adapter "basic" Begin() with 3 workers
trace git-lfs: xfer: adapter "basic" started
trace git-lfs: xfer: adapter "basic" End()
trace git-lfs: xfer: adapter "basic" worker 1 starting
trace git-lfs: xfer: adapter "basic" worker 1 waiting for Auth
trace git-lfs: xfer: adapter "basic" worker 0 starting
trace git-lfs: xfer: adapter "basic" worker 0 stopping
trace git-lfs: xfer: adapter "basic" worker 2 starting
trace git-lfs: xfer: adapter "basic" worker 2 waiting for Auth
trace git-lfs: xfer: adapter "basic" worker 2 auth signal received
trace git-lfs: xfer: adapter "basic" worker 2 stopping
trace git-lfs: xfer: adapter "basic" worker 1 auth signal received
trace git-lfs: xfer: adapter "basic" worker 1 stopping
trace git-lfs: xfer: adapter "basic" stopped
Error downloading object: Crash_Steel_Pipe_Rolling.wav (891a4bd6cf8e6fb0dfc716f7a31219c970dc55d8220e7570b42b328e7edebc9f)
 
Errors logged to C:\Users\a\Documents\test\.git\lfs\objects\logs\20170815T160331.7697439.log
Use `git lfs logs last` to view the log.
error: external filter 'git-lfs filter-process' failed
fatal: Crash_Steel_Pipe_Rolling.wav: smudge filter lfs failed
 
a@ACER-E3-112-C6XG MINGW64 ~/Documents/test (master)
```

**Comment by slonopotamus** [comment]
Did you manage to fix the issue?
What GitLab version this happens with?
Can you provide your git-as-svn.conf?

**Comment by aologos** [comment]
No, I didn't. I stopped to use git-as-svn more than half a year ago, so I can't provide git-as-svn.conf and GitLab version.

**Comment by slonopotamus** [comment]
I think everything will be fine when we fix #175. Closing this issue, nothing to do here.
